src/components/TalentsListPage/TalentsListPage.jsx

```diff
@@ -1,6 +1,14 @@
 import React from "react";
 import { Layout } from "../Layout";
-
+import { SearchPanel } from "../TalentsListPage/SearchPanel/SearchPanel";
+import { Filter } from "./Filter";
+import { TalentsList } from "./TalentsList/TalentsList";
 export function TalentsListPage() {
-	return <Layout></Layout>;
+    return (
+        <Layout>
+            <SearchPanel />
```

Comment:
You better add grid-container for your components with appropriate classname, and set styles with "display: grid;" rule to make appropriate positioning. It is better to use a grid container here, because we need to use positioning along two axes at the same time.
```suggestion
         <div classname={styles["grid-container"]}>
            <SearchPanel />
```